homeassistant/components/garmin_connect/sensor.py

```diff
@@ -165,12 +165,16 @@ def device_class(self):
             return
 
         data = self._data.data
-        if "Duration" in self._type and data[self._type]:
-            self._state = data[self._type] // 60
-        elif "Seconds" in self._type and data[self._type]:
-            self._state = data[self._type] // 60
-        else:
-            self._state = data[self._type]
+        try:
+            if "Duration" in self._type and data[self._type]:
+                self._state = data[self._type] // 60
+            elif "Seconds" in self._type and data[self._type]:
+                self._state = data[self._type] // 60
+            else:
+                self._state = data[self._type]
+        except KeyError:
+            _LOGGER.info("Entity type %s not found in fetched data.", self._type)
```

Comment:
Consider lowering log level to debug.

We don't end logging messages with period.

Reply:
Fixed both issues. thanks!